docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+services:
```

Comment:
Could you please add a file level comment about the purpose of this file (i.e. local dev...)

Reply:
Some comments added and the CONTRIBUTING.md file has been updated accordingly